src/renderer/helpers/chainHelper.ts

```diff
@@ -18,6 +18,13 @@ export const getChainAsset = (chain: Chain): Asset => {
       return AssetAtom
     case 'POLKA':
       return AssetDOT
+    case 'BCH': {
+      // @todo add BCH support
+      return AssetBTC
```

Comment:
as bch stuff is not completed yet at the `xchain-*` im not sure what is the best way to set default here. suggestions are welcome

Reply:
`{ chain: BCHChain, symbol: 'BCH', ticker: 'BCH' }` + add a comment to link it to https://github.com/xchainjs/xchainjs-lib/issues/206